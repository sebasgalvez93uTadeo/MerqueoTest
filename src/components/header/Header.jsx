import React from 'react';

import './styles/style.scss';
import iconMenu from './images/iconMenu.svg';

const Header = () => {
    return ( 
        <header>
            <h1 id="titleMerqueo">Merqueo Test</h1>
            <img src={iconMenu} alt="icon Menu" />
        </header>
     );
}
 
export default Header;