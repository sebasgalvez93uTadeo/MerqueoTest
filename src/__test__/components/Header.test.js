import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Header from '../../components/header/Header';

describe('<Header />', () => {
  const header = mount(<Header />);
  test('Render del componente Footer', () => {
    expect(header.length).toEqual(1);
  });
  test('Render del titulo', () => {
    expect(header.find('#titleMerqueo').text()).toEqual('Merqueo Test');
  });
});
describe('Header Snapshot', () => {
  test('Comprobar UI del componente Header', () =>{
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  });
});