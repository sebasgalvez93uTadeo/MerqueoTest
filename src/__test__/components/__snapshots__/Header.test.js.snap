// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot Comprobar UI del componente Header 1`] = `
<header>
  <h1
    id="titleMerqueo"
  >
    Merqueo Test
  </h1>
  <img
    alt="icon Menu"
    src={Object {}}
  />
</header>
`;
